### DISCUSSION TITLE
goto definition get error: [telescope.builtin.lsp_definitions]: No LSP Definitions found in *.vue files

### DISCUSSION THREAD

**Discussion by ghost** [LazyVim/LazyVim#4730] [Q&A]
In a `*.vue` file, I am unable to navigate to component and function definitions in the script section; however, in a `*.ts` file, goto definition is working fine.

Is the `vue-language-server` not supporting goto definition or do I need to some settings?

https://github.com/user-attachments/assets/49865a6a-b9a2-4489-a4f2-58d56fb7c5db

![image](https://github.com/user-attachments/assets/909273c7-8870-4a63-95a1-61e7e063a017)

**Comment by ghost** [comment]
it's solved, just add lsp config in `init.lua`, but I don't know why if I create `config/lspconfig.lua`, then put code in there, it's not working.
```lua
local lspconfig = require("lspconfig")

lspconfig.volar.setup({
  filetypes = { "typescript", "javascript", "javascriptreact", "typescriptreact", "vue" },
  init_options = {
    vue = {
      hybridMode = false,
    },
  },
})
```

**Reply by lucienlugeek** [reply]
I add the same config into init.lua, but yet not working. Could you plz tell me more details that how to config it right to work.
